### PR TITLE
Added files

### PR DESCRIPTION
**Pull request by TheeDee6**
Added files

### CODE REVIEW

File: controllers/author_controller.js
Comment (TheeDee6) [inline controllers/author_controller.js:1]: Great job

**Review by TheeDee6 — COMMENTED** [review]
nice work

**Review by francesc-pablo — COMMENTED** [review]
solid

**Review by francesc-pablo — COMMENTED** [review]
hm

**Review by Aky3baby — APPROVED** [review]
nice job

**Pull request merged**